examples/README.md

```diff
@@ -0,0 +1,279 @@
+ENCLAVE_SO_PATH=examples/helloworld/_build/lib/
+
+CC_VERS=0
+num_rounds=3
+
+helloworld_test() {
+    # install, init, and register (auction) chaincode
+    try ${PEER_CMD} chaincode install -l fpc-c -n ${CC_ID} -v ${CC_VERS} -p ${ENCLAVE_SO_PATH}
+    sleep 3
+
+    try ${PEER_CMD} chaincode instantiate -o ${ORDERER_ADDR} -C ${CHAN_ID} -n ${CC_ID} -v ${CC_VERS} -c '{"args":["init"]}' -V ecc-vscc
+    sleep 3
+
+    try ${PEER_CMD} chaincode invoke -o ${ORDERER_ADDR} -C ${CHAN_ID} -n ${CC_ID} -c '{"Args":["setup", "ercc"]}' --waitForEvent
+
+    try ${PEER_CMD} chaincode query -o ${ORDERER_ADDR} -C ${CHAN_ID} -n ${CC_ID} -c '{"Args":["getEnclavePk"]}'
+
+    say "do hello world"
+    for (( i=1; i<=$num_rounds; i++ ))
+    do
+        # submit data $i for "asset1"
+        try ${PEER_CMD} chaincode invoke -o ${ORDERER_ADDR} -C ${CHAN_ID} -n ${CC_ID} -c '{"Args":["[\"putData\",\"asset1\",\"'$i'\"]", ""]}' --waitForEvent
+        
+	# get data for "asset1";  should be $i;
+        try ${PEER_CMD} chaincode invoke -o ${ORDERER_ADDR} -C ${CHAN_ID} -n ${CC_ID} -c '{"Args":["[\"getData\",\"asset1\"]", ""]}' --waitForEvent
+    done
+}
+
+# 1. prepare
+para
+say "Preparing Helloworld Test ..."
+# - clean up relevant docker images
+docker_clean ${ERCC_ID}
+docker_clean ${CC_ID}
+
+trap ledger_shutdown EXIT
+
+
+para
+say "Run helloworld  test"
+
+say "- setup ledger"
+ledger_init
+
+say "- helloworld test"
+helloworld_test
+
+say "- shutdown ledger"
+ledger_shutdown
+
+para
+yell "Helloworld test PASSED"
+
+exit 0
+```
+
+Execute:  
+```
+cd FPC-INSTALL-DIR/integration
+make helloworld_test
+```
+
+Output of transaction invocations will look like: 
+```
+...
+2019-08-16 20:38:54.261 UTC [chaincodeCmd] chaincodeInvokeOrQuery -> INFO 002 Chaincode invoke successful. result: status:200 payload:"{\"ResponseData\":\"T0s=\",\"Signature\":\"MEUCIQCMBc1xibInjO4Dc4Ti9w6eXetVDc7G6l6c74NuVanO2wIgZiL1LXF0/EuUZi8y9osXq/1dIvPO1AEgh4fIThB3pX4=\",\"PublicKey\":\"MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEfKNgCOQLI7lawzyWInbdXzyT+qE889vgyf7bFrx0yVZ+XlyOIrMRgFIxvpeUHxystUcJbfPlx85fv6zfsSybkA==\"}" 
+2019-08-16 20:38:56.539 UTC [chaincodeCmd] ClientWait -> INFO 001 txid [6ef1b31334ae727ab15da09f485c6c6455800a82397863fc955259bfabbb753e] committed with status (VALID) at 
+2019-08-16 20:38:56.539 UTC [chaincodeCmd] chaincodeInvokeOrQuery -> INFO 002 Chaincode invoke successful. result: status:200 payload:"{\"ResponseData\":\"eyJuYW1lIjoiYXNzZXQxIiwidmFsdWUiOjN9\",\"Signature\":\"MEYCIQDzHPRaPULt/udhxseFZca3Pu/B4cszfgdW6gHvGXhXUAIhAPMxragYhCFqnJWdNzW8l4vxCVlpzeGjyfpppR98OlW/\",\"PublicKey\":\"MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEfKNgCOQLI7lawzyWInbdXzyT+qE889vgyf7bFrx0yVZ+XlyOIrMRgFIxvpeUHxystUcJbfPlx85fv6zfsSybkA==\"}" 
+helloworld_test.sh: - shutdown ledger
+
+
+ helloworld_test.sh: Helloworld test PASSED 
+ ...
+```
+
+Response from the transactions is marked by `Response Data` and it is base64 encoded. 
```

Comment:
maybe present the payload as pretty json and give and elaborate what we get back here. ResponseData is base64 encoded; the signature and enclave public key.